extensions/cache/runtime/src/main/java/io/quarkus/cache/runtime/CachedResultsDiff.java

```diff
@@ -0,0 +1,28 @@
+package io.quarkus.cache.runtime;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.PARAMETER;
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import jakarta.inject.Qualifier;
+
+import io.quarkus.cache.CachedResults;
+import io.smallrye.common.annotation.Experimental;
+
+/**
+ *
```

Comment:
I think we should document that this is an internal qualifier, it is an implementation mechanism of `@CachedResults` and should not be treated as public type.